### PR TITLE
Finder - resources and selector generator

### PR DESCRIPTION
**Pull request by blakebyrnes**
This PR addresses a few things:
1. The finder has been styled and made functional. You can search resources and generate query selectors2. 
2. QuerySelectors are generating and finding top matches. These are not saved anywhere yet, but shouldn't be much work to do so when we're ready. We can also pretty easily add a "boundary" element or configure options to turn off. Those are being passed back to the client, but there's not ui to pass them around yet. From testing, we might want this sooner than later.
3. Resources are able to be searched for the current url, or a selected range on the slider. The body and url are searched for matches and the results are highlighted in the response body.

Other things:
1. Some focus issues were resolved for ChromeAlive hiding the toolbar (first use was blurring, and you could lose focus dragging timeline)  
2. You can now develop chrome alive UIs without relaunching the app. To do so, run `yarn watch` in chromealive-ui, chromealive-extensions and now in boss. Boss will copy over the new files. You can hit refresh from the devtools panel to reload.

### CODE REVIEW

File: apps/chromealive-core/injected-scripts/generateSelectorMap.ts [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+      console.warn('Selector already tried', selector, {
+        rankSum,
+        layerBitmasks,
+        selectorPermutations,
+      });
+    counter += getSelectorCounts(context, layerBitmasks, selector, rankSum);
+
+    if (counter > nextBreakpoint) {
+      nextBreakpoint += 10e3;
+      console.log('Checking for matches after %s permutations', counter, context.selectors.length);
```
Comment (calebjclark) [inline apps/chromealive-core/injected-scripts/generateSelectorMap.ts:99]: Should this console.log be here?

File: apps/chromealive-core/lib/hero-plugin-modules/ElementsModule.ts [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import { IPuppetPage } from '@ulixee/hero-interfaces/IPuppetPage';
 import highlightConfig from './highlightConfig';
 import HeroCorePlugin from '../HeroCorePlugin';
+import * as fs from 'fs';
+import { ISelectorMapContext } from '../../injected-scripts/generateSelectorMap';
```
Comment (calebjclark) [inline apps/chromealive-core/lib/hero-plugin-modules/ElementsModule.ts:5]: Defined but never used

File: apps/chromealive-core/injected-scripts/generateSelectorMap.ts (hunk 1 above)
Comment (blakebyrnes) [reply apps/chromealive-core/injected-scripts/generateSelectorMap.ts:99]: I can remove it, but I was finding it useful to keep an eye on.

Comment (calebjclark) [reply apps/chromealive-core/injected-scripts/generateSelectorMap.ts:99]: Ok. We can keep it in for now... at some point we should probably push this to the UI.

**Pull request merged**